save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestExecutionController.kt

```diff
@@ -49,6 +49,7 @@ class TestExecutionController(private val testExecutionService: TestExecutionSer
      * @return response
      */
     @PostMapping(value = ["/saveTestResult"])
+    @Transactional
```

Comment:
Why do we need transactional here? It seems that this controller is not affected in this PR anywhere else

Reply:
Yes, but I changed tests for this request and forgot to remove it.